courses/fundamentals_of_ada/labs/answers/420_ada_finalization.txt

```diff
@@ -23,6 +23,7 @@ package body Keys_Pkg is
    Global_In_Use : array (Character range 'a' .. 'z') of Boolean :=
      (others => False);
 
+   pragma Warnings ( Off );
```

Comment:
We'll need to kick pragma out of examples when they are included, that's a whole can of worms to open otherwise

Reply:
I think generally you're correct, but this particular module is not going to be used often, so I'll leave it for now